src/test/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpointUnitTest.java

```diff
@@ -125,4 +128,19 @@ public void verifyBadBindingResultThrowsException() throws Exception {
         sampleEndpoint.createSampleUnitRequest(null, bindingResult);
     }
 
+    @Test
+    public void ensureAttributesReturnedById() throws Exception {
+
+//      UUID id = UUID.randomUUID();
+//      SampleAttributes sampleAttribs = new SampleAttributes();
+//      Map<String, String> attribs = new HashMap<>();
+//
+//      attribs.put("ref", id.toString());
+//      sampleAttribs.setAttributes(attribs);
+//
+//      when(sampleService.findSampleAttributes(any())).thenReturn(sampleAttribs);
+//
+//      sampleEndpoint.requestSampleAttributes(id);
+    }
```

Comment:
Entire test is commented out, should be removed or implemented.

Reply:
I've added a unit test which will cover getting sample attributes from a sample unit